caikit/runtime/service_factory.py

```diff
@@ -187,7 +189,12 @@ def get_service_package(
             )
             service_name = f"{ai_domain_name}Service"
         else:  # service_type == cls.ServiceType.TRAINING
-            rpc_list = service_generation.create_training_rpcs(clean_modules)
+            output_target_type = make_output_target_message(
+                # plugin_factory=OutputTargetPluginFactory
```

Comment:
What's this about?

Reply:
ah, just a leftover bit I meant to delete. But also... do we want to pass in some extra config here so users can re-name / re-number these oneof fields?